src/framework/execution/__tests__/logger.test.ts

```diff
@@ -0,0 +1,168 @@
+import { createIntegrationLogger } from '../logger';
+import Logger from 'bunyan';
+import { IntegrationInvocationConfig } from '../types';
+
+const invocationConfig = {} as IntegrationInvocationConfig;
+const loggerName = 'integration-logger';
+
+describe('logger.trace', () => {
+  test('includes verbose: true for downstream verbose log pruning', () => {
```

Comment:
Just wanted to point out... I'm not sure all the reasons there was a need to add an additional property, but I wonder if eventually we should change our log ingesting code to ignore `level=10` so that we don't have to add this additional property.

Reply:
iirc the decision for the allowing a verbose flag was to give a little more fine grain control over how logs are ignored while still giving services the ability to open the floodgates in the rare times when we wanted lower level logs to be published.

Also I think in the past our pipeline had a few edge cases where `info` logs could become excessively large and for some reason we wanted to keep those logs at that level (probably to still capture that data in CW).